packages/pong-engine/src/game.ts

```diff
@@ -0,0 +1,60 @@
+import { GameBall, GameState } from './models';
+import {
+  CANVAS_HEIGHT,
+  CANVAS_WIDTH,
+  BALL_RADIUS,
+  BRICK_WIDTH,
+} from './constants';
+import { reducer, initialBallState, initialPaddleState } from './state';
+
+const losePoint = (ball: GameBall) => ball.y > 1.5 * CANVAS_HEIGHT;
```

Comment:
I'm not reviewing the math, but I have to say that I don't get how the ball being at a y coordinate larger than 1.5 times the canvas makes it a lose point. It's kind of works so I'm inclined to believe it makes sense, but maybe it's dead code, and I have to voice my question

Reply:
This is a visual “trick” so we have a goal effect. This is, game does not reset immediately after scoring a point and we give a few milliseconds to the player to realize the ball went out of the court